test/system/books_test.rb

```diff
@@ -3,45 +3,68 @@
 require 'application_system_test_case'
 
 class BooksTest < ApplicationSystemTestCase
+  include Devise::Test::IntegrationHelpers
+
   setup do
-    @book = books(:one)
+    sign_in users(:one)
   end
 
   test 'visiting the index' do
     visit books_url
-    assert_selector 'h1', text: 'Books'
+    assert_selector 'h1', text: '本'
   end
 
-  test 'creating a Book' do
+  test 'creating a book' do
     visit books_url
-    click_on 'New Book'
+    click_on '新規作成'
+
+    fill_in 'book[title]', with: 'The Return of the King'
```

Comment:
```suggestion
    fill_in 'タイトル', with: 'The Return of the King'
```

のように、画面上に表示されているラベルを指定できると思います。
この方が実際のブラウザ操作がイメージしやすくなるので良いです。